packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -173,6 +181,37 @@ export async function setSourceBundleUrl(addinId: string, components: SourceBund
   }
 }
 
+export async function setWebView(addinId: string, webViewType: WebViewType | undefined): Promise<void> {
+  const key = getDeveloperSettingsRegistryKey(addinId);
+  switch (webViewType){
+    case undefined:
+    case WebViewType.Default:
+      await registry.deleteValue(key, WebViewSelection);
+      break;
+    case WebViewType.IE:
+    case WebViewType.Edge:
+    case WebViewType.EdgeChromium:
+      const webViewString: string = <string> webViewType;
```

Comment:
I still think it would be better to define string constants for the registry values used and to use them here and in the toWebViewType function.